save-backend/src/main/kotlin/org/cqfn/save/backend/service/ExecutionService.kt

```diff
@@ -13,24 +13,24 @@ import java.time.LocalDateTime
  * Service that is used to manipulate executions
  */
 @Service
-class ExecutionService(private val executionRepository: ExecutionRepository) {
+class ExecutionService(private val baseEntityRepository: BaseEntityRepository<Execution>) {
```

Comment:
Why change the name?

Reply:
I agree, the names can be left the same)